projects/schematics/src/migrations/4_0/removed-public-api-deprecations/removed-public-api-deprecation.ts

```diff
@@ -34,6 +61,97 @@ export const REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
   //   importPath: SPARTACUS_STOREFRONTLIB,
   //   comment: `'${VARIANT_STYLE_SELECTOR_COMPONENT}' was moved to @spartacus/product/variants.`,
   // },
+
+  // projects/assets/src/translations/translation-chunks-config.ts
+  {
+    node: TRANSLATION_CHUNKS_CONFIG,
+    importPath: SPARTACUS_STOREFRONTLIB,
+    comment: `Following translation chunks '${MINI_LOGIN_TRANSLATION_CHUNK}', '${UPDATE_EMAIL_FORM_TRANSLATION_CHUNK}', '${FORGOTTEN_PASSWORD_TRANSLATION_CHUNK}', '${LOGIN_FORM_TRANSLATION_CHUNK}', '${REGISTER_TRANSLATION_CHUNK}' were moved to ${SPARTACUS_USER}.`,
+  },
+  // projects/storefrontlib/src/cms-components/cms-lib.module.ts
+  {
+    node: CMS_LIB_MODULE,
+    importPath: SPARTACUS_STOREFRONTLIB,
+    comment: `Following module imports '${CLOSE_ACCOUNT_MODULE}', '${FORGOT_PASSWORD_MODULE}', '${RESET_PASSWORD_MODULE}', '${UPDATE_EMAIL_MODULE}', '${UPDATE_PASSWORD_MODULE}', '${UPDATE_PROFILE_MODULE}', '${USER_COMPONENT_MODULE}', were removed. Those modules are now part of ${SPARTACUS_USER}.`,
+  },
+  // projects/storefrontlib/src/cms-components/myaccount/close-account/close-account.module.ts
+  {
+    node: CLOSE_ACCOUNT_MODULE,
+    importPath: SPARTACUS_STOREFRONTLIB,
+    comment: `'${CLOSE_ACCOUNT_MODULE}' was moved to ${SPARTACUS_USER_PROFILE_COMPONENTS}.`,
+  },
+  // projects/storefrontlib/src/cms-components/myaccount/close-account/components/close-account/close-account.component.ts
+  {
+    node: CLOSE_ACCOUNT_COMPONENT,
+    importPath: SPARTACUS_STOREFRONTLIB,
+    comment: `'${CLOSE_ACCOUNT_COMPONENT}' was moved to ${SPARTACUS_USER_PROFILE_COMPONENTS}.`,
+  },
+  // projects/storefrontlib/src/cms-components/myaccount/close-account/components/close-account-modal/close-account-modal.component.ts
+  {
+    node: CLOSE_ACCOUNT_MODAL_COMPONENT,
+    importPath: SPARTACUS_STOREFRONTLIB,
+    comment: `'${CLOSE_ACCOUNT_MODAL_COMPONENT}' was moved to ${SPARTACUS_USER_PROFILE_COMPONENTS}.`,
+  },
+  // projects/storefrontlib/src/cms-components/myaccount/forgot-password/forgot-password.module.ts
+  {
+    node: FORGOT_PASSWORD_MODULE,
+    importPath: SPARTACUS_STOREFRONTLIB,
+    comment: `'${FORGOT_PASSWORD_MODULE}' was moved to ${SPARTACUS_USER_PROFILE_COMPONENTS}.`,
+  },
+  // projects/storefrontlib/src/cms-components/myaccount/forgot-password/forgot-password.component.ts
+  {
+    node: FORGOT_PASSWORD_COMPONENT,
+    importPath: SPARTACUS_STOREFRONTLIB,
+    comment: `'${FORGOT_PASSWORD_COMPONENT}' was moved to ${SPARTACUS_USER_PROFILE_COMPONENTS}.`,
```

Comment:
Can we mention about the changes to the component here and in docs (changed html, separated component service, changes in the properties/methods)?